src/raftstore/store/fsm/batch.rs

```diff
@@ -294,20 +293,28 @@ struct Poller<N: Fsm, C: Fsm, Handler> {
 
 impl<N: Fsm, C: Fsm, Handler: PollHandler<N, C>> Poller<N, C, Handler> {
     fn fetch_batch(&self, batch: &mut Batch<N, C>, max_size: usize) {
-        let mut pushed = match self.fsm_receiver.recv() {
-            Ok(fsm) => batch.push(fsm),
-            Err(_) => return,
+        let curr_batch_len = batch.len();
```

Comment:
Please add comments here for this changing.

Reply:
Fixed.